src/frontend/js/modules/search/search.less

```diff
@@ -74,7 +74,7 @@
     margin-top: 15px;
 
     .listview.lv-lg .lv-item {
-      padding: 7px 10px;
+      padding: 7px 10px 40px;
```

Comment:
This change can impact other search results? Contact, users, etct

Reply:
I agree with @tuanlc. I think this can be fixed in esn-frontend-calendar instead.

Reply:
Or find a way to make the search item more "responsive" instead of trying to increase its padding/height/width.